example/golang.mk

```diff
@@ -14,6 +14,7 @@ BUILD_MACHINE=$(shell echo $$HOSTNAME)
 BUILD_USER=$(shell whoami)
 
 BUILD_FLAGS=-ldflags "\
+	-s -w \
```

Comment:
Sorry, we won't strip the symbol tables. They are very useful for debugging.

Reply:
regarding strip sure, i am fine with it, maybe additional conditional is better way in any case

Reply:
question regarding conditional is, do you use gnu make or BSD make (macos i have no clue what exactly have)

Reply:
I think the easiest approach is to introduce a new variable (`ADDITIONAL_LDFLAGS`?), and define this in the `Makefile` and not in `golang.mk`. The Makefile should not fail if the variable isn't set.